azure-iot-device/azure/iot/device/iothub/aio/async_inbox.py

```diff
@@ -70,6 +70,8 @@ async def get(self):
     def empty(self):
         """Returns True if the inbox is empty, False otherwise
 
+        Note that there is a race condition here, and this may not be accurate.
```

Comment:
>Note that there is a race condition here, and this may not be accurate. [](start = 8, length = 71)

please add some text that explains the race condition.  It doesn't need to be too explicit, but it should give people somewhere to look.  "race condition here" implies that the async inbox has a race condition, but I don't think that's the case.  I think you need to say something like "this function doesn't always return an accurate value because a message might arrive while the client is in process of disconnecting.  If the message arrives after the blorg is schmotzed, then the schenzer might shmear without mauering."  Same for the sync version